postinstall.sh

```diff
@@ -5,3 +5,6 @@ SCRIPT_LOGFILE="./logs/"$(basename -s .sh $BASH_SOURCE)".log"
 
 echo "running "$SCRIPT_NAME
 
+#add the file containing the aliases to your 
+cat aliases.sh>>~/.bashrc
```

Comment:
Where's that `aliases.sh` file? You're neither adding a default one nor checking if it exists first before `cat`ting it 😕 That would fail in many cases, because not everybody's got a custom aliases.  
I suggest either you add a default/template `aliases.sh` or check for file existence first. Or maybe you could leave it as is and handle the error (if any) and redirect it to a log file.